### PR TITLE
[MIG] web_widget_child_selector: Migration to 14.0

### PR DESCRIPTION
**Pull request by olgamarcocb**
Standard Migration

### CODE REVIEW

File: web_widget_child_selector/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Use child_selection widget on a field.
```
Comment (etobella) [inline web_widget_child_selector/readme/USAGE.rst]: typo error that comes before the migration. The widget is child_selector. Can you change it please :smile:

**Review by etobella — APPROVED** [review]
LGTM. Just one small comment.

Thanks for the work

File: web_widget_child_selector/readme/USAGE.rst (hunk 1 above)
Comment (olgamarcocb) [reply web_widget_child_selector/readme/USAGE.rst]: Of course! 😉

**Review by albariera — APPROVED** [review]
LGTM :+1:

**Pull request merged**